initScripts/x86_64/WindowsServer_2016/boot.ps1

```diff
@@ -207,6 +207,12 @@ Function boot_reqKick() {
   popd
 }
 
+Function print_summary() {
+  Write-Output "==== Summary ===="
+  Write-Output "- A firewall rule to allow connections on port 2375 was added"
```

Comment:
A firewall rule named "shippable-docker" has been created to allow incoming connections on TCP port 2375.